### ISSUE TITLE
[BUG] Spread syntax inside the `classnames` package crashes ESLint

### ISSUE THREAD

**Issue by Serator** [francoismassart/eslint-plugin-tailwindcss#119]
**Describe the bug**
See the "To Reproduce" section.

**To Reproduce**
1. Write `<div className={classnames({...{'test': true}})} />`
2. See error

**Expected behavior**
There should be no errors.

**Screenshots**
![image](https://user-images.githubusercontent.com/625005/152657744-1c778968-08b7-4831-92dd-65e00663e734.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Softwares + version used:
     -  VSCode: 1.64
     - eslint-plugin-tailwindcss: 3.4.3
     - tailwindcss: 3.0.18

**Comment by francoismassart** [comment]
Hi @Serator,
please try `eslint-plugin-tailwindcss@3.4.4-beta.0`
It'll probably not handle multiple depth of spread object but it may be sufficient for you.

Let me know how it works for you.

**Comment by Serator** [comment]
@francoismassart Yeah, it works, but it causes a problem elsewhere: `<div className={classnames({'test': true})} />`.
```
ESLint stack trace:
TypeError: p.value.split is not a function
Occurred while linting Test\index.tsx:23
Rule: "tailwindcss/no-contradicting-classname"...
```

**Comment by francoismassart** [comment]
Closing this issue, cleaning up before release of v4.
